vpc/service/action_worker.go

```diff
@@ -173,7 +173,7 @@ func (actionWorker *actionWorker) worker(ctx context.Context, wq workqueue.RateL
 			err = errors.Wrap(err, "Cannot set lock timeout to 1000 milliseconds")
 			tracehelpers.SetStatus(err, span)
 			logger.G(ctx).WithError(err).Error()
-			return nil
+			return err
```

Comment:
Is this and the other `return err` the *actual* fix in this PR then?

Reply:
And the Go upgrade. Where we would get into a rapid leader-reelection cycle because it'd keep picking up bad txns.

Reply:
Oh right right.